### ISSUE TITLE
Detectron keypoints prediction puts some annotations in wrong image.

### ISSUE THREAD

**Issue by ArchNew** [roytseng-tw/Detectron.pytorch#143]
### Expected results
![fda91f0ba76081c65535e3ffea30145](https://user-images.githubusercontent.com/11910381/45012879-b6295d00-b04b-11e8-86e8-ee3fe2dcfaa2.png)
I expect the cyan skeletion in this image, this person.

### Actual results
![6e10dd97c5803759c75a3e74cc63371](https://user-images.githubusercontent.com/11910381/45012896-cb9e8700-b04b-11e8-8678-b842072b41e2.png)
The Detectron's output puts the cyan skeletion in a neighboring picture...which results a very low detection rate on validation set.

### Detailed steps to reproduce

I try to use detectron on posetrack's validation set [https://posetrack.net]. Unfortunately, it doesn't always put its predictions in the right picture. It sometimes puts them in a neighboring one, as I demonstrated above.

I wrote the code responsible for converting posetrack validation set's annotations into coco format. And I double checked that every image has its unique image_id. There's no two image use the same image_id. I don't think validation set's ground truth matters here, but I also double checked that the ground truth is the same before and after the conversion.  

### System information

* Operating system: Ubuntu 16.04
* CUDA version: 8.0
* cuDNN version: 7.2.1
* GPU models (for all devices): Titan XP
* python version: 2.7.13
* pytorch version: whatever version of caffe2 Detectron runs on.
* Anything else that seems relevant: ?

**Comment by ArchNew** [comment]
It's my bad. My conversion is wrong.
